.github/pull_request_template.md

```diff
@@ -9,16 +9,49 @@
 
 ## Issue type checklist
 <!--- What type of change are you submitting? Check the boxes that apply: -->
-- [ ] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
-- [ ] **New feature**: non-breaking change that adds functionality.
-- [ ] **Breaking change**: fix or feature that will cause current functionality to change.
+
+- [] **Bug fix**: non-breaking change which fixes an issue, link to the issue above.
```

Comment:
Just curious on everyone's thought where I removed a space inside [] so that it's just an `x` that gets added without the need for deleting a space as well.

Reply:
Fine with me.